tests/integration/standard/test_metadata.py

```diff
@@ -501,7 +501,7 @@ def test_indexes(self):
         self.assertIn('CREATE INDEX e_index', statement)
 
     @greaterthancass21
-    @requires_collection_indexes
+    @xfail_scylla_version('failing cause of scylladb/scylladb#19278', oss_scylla_version='6.1', ent_scylla_version='2025.1')
```

Comment:
I'm not very familiar with enterprise release process, backporting etc. Are we sure that the fix won't be included in any 2024.x version?